content/en/docs/concepts/policy/limit-range.md

```diff
@@ -1,64 +1,64 @@
 ---
 reviewers:
 - nelvadas
-title: Limit Ranges
```

Comment:
The original title, “Limit Ranges”, should localize into target languages. I like that it would.
How about changing the title to “Resource Limit Ranges”?

Reply:
I guess I would prefer to just keep `Limit Ranges` in this case, but I agree on the rationale.